Podfile

```diff
@@ -0,0 +1,16 @@
+platform :ios, '10.0'
+
+target 'ISICkonto' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for ISICkonto
+	pod 'SnapKit', '~> 4.2'				#constraints
+	pod 'Kanna', '~> 4.0'				#html parser
+	pod 'RxAlamofire', '~> 4.3'			#reactive wrapper to Alamofire for http networking
+	pod 'RxSwift', '~> 4.4'
+	pod 'RxCocoa', '~> 4.4'				
+	pod 'NSObject+Rx'
```

Comment:
doporučuju dávát omezení verze u každé knihovny, je to víc práce, ale mnohem bezpečnější 😊